TOOLS/gen-x11-icon.sh

```diff
@@ -4,8 +4,8 @@
 # etc/mpv-icon...), and it will write video/out/x11_icon.bin.
 
 conv() {
-    echo
-    echo `identify -format "icon: %w %h" "$1"`
+    printf "\n"
```

Comment:
Do you want me to do that?
I thought this way it looked a bit cleaner, but I can change.